openedx/core/djangoapps/credentials/utils.py

```diff
@@ -55,7 +55,7 @@ def get_user_program_credentials(user):
     programs_credentials = []
     for credential in credentials:
         try:
-            if 'program_id' in credential['credential'] and credential['status'] == 'awarded':
+            if 'program_uuid' in credential['credential'] and credential['status'] == 'awarded':
```

Comment:
Your method, `get_user_credentials`, needs to be updated to call the `credentials` endpoint for API v2.